code/extension/Member.php

```diff
@@ -26,6 +26,8 @@ class Member extends \DataExtension
 
     private static $admins_can_disable = false;
 
+    private static $regenerate_on_activation = true;
```

Comment:
Would that also be leading to another feature? That if the token doesn't regen between activations they could opt to click a regenerate button as a force?